builtin/v10/eam/eam_types.go

```diff
@@ -0,0 +1,27 @@
+package eam
+
+import "github.com/filecoin-project/go-address"
+
+type CreateParams struct {
+	Initcode []byte
+	Nonce    uint64
+}
+
+type Create2Params struct {
+	Initcode []byte
+	Salt     [32]byte
+}
+
+type CreateAccount struct {
+	// Pubkey is the secp256k1 public key
+	Pubkey [65]byte
```

Comment:
Can we define a SECP_PUBKEY_LENGTH or something somewhere?

Reply:
surely.